bin/ch/WScriptJsrt.cpp

```diff
@@ -3,6 +3,7 @@
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 #include "stdafx.h"
+#include <vector>
```

Comment:
Do we already depend on `<vector>` anywhere else? If not, do we already have an implementation of a vector-like object in ChakraCore that you could depend on?